ark-testing/src/util.rs

```diff
@@ -19,3 +13,34 @@ pub fn init_logging() -> anyhow::Result<()> {
 		.try_init();
 	Ok(())
 }
+
+/// Returns the directory where all test data will be written
+///
+/// By default this is written in the `./test` directory at the project root.
+/// You can also set TEST_DIRECTORY to pick another location.
+/// You are responsible to ensure the `TEST_DIRECTORY` exists
+pub fn test_data_directory() -> PathBuf {
+	match std::env::var_os(TEST_DIRECTORY) {
+		Some(directory) => { PathBuf::from(directory) },
+		None => {
+			let path = get_cargo_workspace().join("test");
```

Comment:
You have `test` here but `tests` in the gitignore.

Reply:
Fixed in 2675469b90e46f7d4564e705778c9f6280a7fb6e.

I've changed the `.gitignore` file